diff --git a/libFDK/include/common_fix.h b/libFDK/include/common_fix.h
index 8d0dfde33a134db8ef15ad401123372875305079..d74293b4c9c8c4d2d11e1b07047bd1c0978f29c3 100644
--- a/libFDK/include/common_fix.h
+++ b/libFDK/include/common_fix.h
@@ -406,7 +406,7 @@ FDK_INLINE FIXP_SGL fMax(FIXP_SGL a, FIXP_SGL b) { return fixmax_S(a, b); }
 #if !defined(__LP64__)
 FDK_INLINE INT fMax(INT a, INT b) { return fixmax_I(a, b); }
 FDK_INLINE INT fMin(INT a, INT b) { return fixmin_I(a, b); }
-#if !defined(_MSC_VER) && defined(__x86_64__)
+#if !defined(_MSC_VER) && defined(__x86_64__) && !defined(_WIN32)
 FDK_INLINE SHORT fMax(SHORT a, SHORT b) { return fixmax_S(a, b); }
 FDK_INLINE SHORT fMin(SHORT a, SHORT b) { return fixmin_S(a, b); }
 #endif
